-include ../Makefile.credentials
include ../Makefile.deploytemplate

WORKFLOW_NAME=magi2
TOOL_FOLDER_NAME=magi2
WORKFLOW_VERSION=release_30
WORKFLOW_DESCRIPTION="<img src="https://magifiles.nersc.gov/static/img/MAGI_logo.png" width="400"><br>MAGI 2<hr><a href='https://doi.org/10.1021/acschembio.8b01107'>Onur Erbilgin, Oliver Rubel, Katherine B Louie, Matthew Trinh, Markus DeRaad, Tony Wildish, Daniel Udwary, Cindi Hoover, Sam Deutsch, Trent R Northen, Benjamin P Bowen. MAGI: A Method for Metabolite Annotation and Gene Integration., ACS Chem Biol. 2019 Apr 19;14(4):704-714. doi: 10.1021/acschembio.8b01107. DOI: 10.1021/acschembio.8b01107 Epub 2019 Apr 4. PMID: 30896917.</a> <hr> \
						MAGI was built to make connecting metabolomics data with genes easier for researchers. Metagenomics and single-cell sequencing have enabled glimpses into the vast metabolic potential of Earth’s collective biological systems. Yet, for the most part we can’t accurately predict nor identify the products of most biosynthetic pathways. Most of what we know of microbial biochemistry is based on characterization of a few model microorganisms, and these findings have been extended through sequence correlations to the rest of sequence space. Unfortunately, these extrapolations have questionable validity for the vast majority of environmental microbes. Connecting metabolomics observations with genomic predictions is crucial to overcome the limitations of each and to strengthen the biological conclusions made by both. MAGI provides a fundamentally different approach for directly linking novel sequences to their biochemical functions and products."
